plugins/modules/cohesity_migrate_vm.py

```diff
@@ -560,15 +607,16 @@ def main():
             endpoint=dict(type="str", required=True),
             environment=dict(choices=["VMware"], default="VMware"),
             job_vm_pair=dict(type="dict", required=True),
-            datastore_name=dict(type="str", default=""),
+            datastore_name=dict(type="str", required=True),
```

Comment:
Datastore name is required ? This breaks compatibility ?

Reply:
Resource pool and datastore name are required parameters.

Reply:
But Datastore isn't required in UI correct ?